README.md

```diff
@@ -378,6 +378,42 @@ $ cargo test
 
 from the repository root.
 
+### Tips
+
+#### Windows Powershell
+
+##### Setup function alias
+
+Often you can find a need to make alias for the favourite utility.
+
+But powershell function aliases do not behave like your typical linux shell alias.
+
+You always need to propagate arguments and **Stdin** input.
+But it cannot be done simply as `function grep() { $input | rg.exe --hidden $args }`
+
+Use below example as reference to how setup alias in powershell.
+
+```powershell
+function grep {
```

Comment:
I think this looks OK to me. Is there any way to put this function into a file that is automatically loaded whenever powershell starts?

cc @retep998